src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy

```diff
@@ -72,6 +72,30 @@ class AutoRerouteV2Spec extends HealthCheckSpecification {
         database.resetCosts()
     }
 
+    def "Single switch flow changes status on switch up/down events"() {
+        given: "single switch flow"
+        def sw = topology.getActiveSwitches()[0]
+        def flow = flowHelperV2.singleSwitchFlow(sw)
+        northboundV2.addFlow(flow)
+        Wrappers.wait(WAIT_OFFSET) {
+            assert northboundV2.getFlowStatus(flow.flowId).status == FlowState.UP
+        }
+        when: "Switch become disconnected"
+        def blockData = switchHelper.knockoutSwitch(sw, mgmtFlManager)
+        then: "Flow changes status to 'Down'"
+        Wrappers.wait(WAIT_OFFSET) {
+            assert northboundV2.getFlowStatus(flow.flowId).status == FlowState.DOWN
+        }
+        when: "Switch wend back online"
+        switchHelper.reviveSwitch(sw, blockData, true)
+        then:
```

Comment:
Add description so that steps form a valid test case in the end.
Please add a blank line before when/then to keep the style similar to other tests